### DISCUSSION TITLE
External libraries failing…..

### DISCUSSION THREAD

**Discussion by Bobski2022** [immich-app/immich#9795] [Q&A] [answered]
Tried to add the external library, the import library. But get this when entering it in,
![IMG_1221](https://github.com/immich-app/immich/assets/113545157/781dba16-c0d0-4489-96b6-cae1f8b0c3be)
If I enter in /mnt/user/iphoto/ or photo it still has the error?
In the set up:
![IMG_1219](https://github.com/immich-app/immich/assets/113545157/da1f6595-82e0-4253-81c6-4d8a236b09b6)
This matches where the photo for import are?

Really strange, but any help would be appreciated 
Thank you
Dave

**Comment by martabal** [comment, marked as answer]
The path in Immich should be `/import`

**Reply by Bobski2022** [reply]
Hahahhha thank you… this worked. I’m still learning, this is all new.
Thanks martabal appreciate that
